packages/paste-core/components/account-switcher/src/AccountSwitcherBadge.tsx

```diff
@@ -16,6 +16,8 @@ const AccountSwitcherBadge = React.forwardRef<HTMLButtonElement, AccountSwitcher
   }
 );
 
+// omit variant from MenuBadgePropTypes because it conflicts with AccountSwitcherBadge
+const {variant, ...accountSwitcherPropTypes} = MenuBadgePropTypes;
```

Comment:
question: No error for variant being unused?

Reply:
We set it in the component above. AccountSwitcher is a subset of Menu, So I want to reuse the proptypes from Menu as much as I can, but because _we_ set variant inside account switcher, the user can't so I don't want to error for them